decidim-proposals/spec/services/decidim/proposals/proposal_serializer_spec.rb

```diff
@@ -8,7 +8,7 @@ module Proposals
       subject do
         described_class.new(proposal)
       end
-
+      let!(:body) { "This is a simple proposal body example." }
```

Comment:
Why are we changing how the body is generated? Using faker is better almost all the times IMO